test/jit/test_freezing.py

```diff
@@ -40,48 +40,58 @@ def test_freeze_module(self):
         class M(nn.Module):
             def __init__(self):
                 super(M, self).__init__()
-                self.a = 1         # folded
```

Comment:
Can we also add some unit tests that test the new serialization behavior in isolation?